protocol/otlp/decoder_test.go

```diff
@@ -53,7 +53,7 @@ func TestDecoder(t *testing.T) {
 			var msg metricsservicev1.ExportMetricsServiceRequest
 			msg.ResourceMetrics = []*metricsv1.ResourceMetrics{
 				{
-					InstrumentationLibraryMetrics: []*metricsv1.InstrumentationLibraryMetrics{
+					ScopeMetrics: []*metricsv1.ScopeMetrics{
```

Comment:
While `InstrumentationLibraryMetrics` is deprecated, there's guidance in https://github.com/open-telemetry/opentelemetry-proto-go/blob/main/otlp/metrics/v1/metrics.pb.go#L309 on converting `InstrumentationLibraryMetrics` to their equivalent `ScopeMetrics`. Or do you plan for this to be handled in a separate diff?

Reply:
We are actually already using the commit that changed it in this branch https://github.com/signalfx/ingest-protocols/tree/otlp.  I'll split out the commit to a separate PR once the lint stuff merges.

I wasn't aware there was guidance to keep using InstrumentationLibraryMetrics but that would only apply for people that upgrade the library and don't change the field which shouldn't really be an issue for collector traffic or official instrumentation that we support so I'm not too worried about it.

Reply:
I didn't know about the usage patterns. Thanks for the explanation.